RELEASE.md

```diff
@@ -24,11 +24,13 @@ The next sections include detailed instructions on how to complete each step of
 
 ## 1. Deciding to release a new version
 
--   [ ] 1.  Inform the engineering team that you are the release manager for a new release in the Slack channel #team-engineering (tag @engineers) and get a status from each squad on any changes that may be on the master branch of components that may be deal breakers for the release.
+-   [ ] 1.  Inform the engineering team that you are the release manager for a new release in the Slack channel [#team-engineering](https://codacy.slack.com/channels/team-engineering) (tag @engineers) and get a status from each squad on any changes that may be on the master branch of components that may be deal breakers for the release.
 
--   [ ] 2.  Confirm with every squad if the release can go ahead.
+-   [ ] 2.  Inform all stakeholders outside the engineering team that a new release proccess is being started, in the Slack channel [#sh_releases](https://codacy.slack.com/channels/sh_releases) (tag @here).
```

Comment:
I'm actually not sure which channel we are supposed to use. Is it enterprise-releases or sh_releases?

Reply:
From what I understand, we sometimes use the channel #enterprise_releases for discussing the progress on releasing a new version, and the channel #sh_releases is more used for the automatic notifications from CircleCI.

Here are the topics for the channels:

- #enterprise_releases

    `Owner: @Ivo; Purpose: Discuss technical details for enterprise releases. Enterprise Release Process https://github.com/codacy/chart/blob/master/RELEASE.md`

- #sh_releases

    `Owner: @joao.costa; Purpose: Automatic notifications whenever there is a new self-hosted release`

This separation can be useful so that people who are only interested on knowing when there is a new release available (but who do not want to participate in the discussion that happens beforehand) only join the #sh_releases channel.

Reply:
cc @gjsduarte @paulopontesm

Reply:
@pauloribeiro-codacy I was also thinking that was the logic behind it. I will update accordingly